VirtualReality/Widgets/qMRMLVirtualRealityView.h

```diff
@@ -100,14 +100,51 @@ class Q_SLICER_QTMODULES_VIRTUALREALITY_WIDGETS_EXPORT qMRMLVirtualRealityView :
   /// Get underlying RenderWindow
   Q_INVOKABLE bool isHardwareConnected()const;
 
+  //--------------------------------------------------
+  // Hacks to limit interaction
```

Comment:
Would it be possible to clarify this ?

Reply:
Maybe we could call these functions "Utility functions for convenient setting of frequently used configurations" ? @drouin-simon